### PR TITLE
Install Oracle required packages for RHEL8

### PR DESCRIPTION
**Pull request by jcnars**
Closing the loop on RHEL8 compatibility certification with bms-toolkit.

Internal reference: https://b.corp.google.com/issues/239771476#comment52

### CODE REVIEW

File: roles/ora-host/tasks/main.yml [hunk 1]
```diff
@@ -97,6 +97,17 @@
     - oraclelinux_repo.stat.exists|bool
   tags: os-packages
 
+- name: Install Oracle required packages (base/non-rhui config) for RHEL8
+  yum:
+    name: "{{ oracle_required_rpms_el8 }}"
+    state: present
+    lock_timeout: 180
+    enablerepo: rhel-8-server-optional-rpms
```
Comment (mfielding) [inline roles/ora-host/tasks/main.yml:105]: Checking an EL8 server I don't see this repo.  https://access.redhat.com/solutions/5331391 seems to indicate that the extras and optional repos no longer exist in RHEL8, and that the packages have been integrated into the base.

repoquery seems to confirm this:  all the packages in `oracle_requires_rpms_el8` exist in either `rhel-8-for-x86_64-appstream-rpms` or `rhel-8-for-x86_64-baseos-rpms`

```bash
[mfielding@at-4633612-svr002 ~]$ sudo repoquery -q --qf='%{repoid}' $(cat el8-rpms)
rhel-8-for-x86_64-appstream-rpms
rhel-8-for-x86_64-baseos-rpms
```

So I think this line is actually unnecessary

Comment (jcnars) [reply roles/ora-host/tasks/main.yml:105]: Thanks for the details.
Sending you the fixed code for review.

**Comment by jcnars** [comment]
Code ready for review.

**Review by mfielding — APPROVED** [review]

**Pull request merged**